src/test/java/com/qaprosoft/carina/demo/mfp/MobileTest.java

```diff
@@ -125,7 +125,15 @@ public void customDashboardTest() {
         softAssert.assertTrue(commonPage.isItemByTextPresent(IConstants.CALORIES),
                 IConstants.CALORIES + " label is not present");
 
-        diaryPage.selectCustomDashboard(CustomDashboardPageBase.Options.CUSTOM);
+        CustomSummaryPageBase customSummaryPage = (CustomSummaryPageBase) diaryPage.selectCustomDashboard(
+                CustomDashboardPageBase.Options.CUSTOM);
+        softAssert.assertTrue(customSummaryPage.isNutrientChecked(CustomSummaryPageBase.Nutrients.FAT),
+                CustomSummaryPageBase.Nutrients.FAT + " nutrient isn't checked");
```

Comment:
you do not have method checkNutrient? why? you assumed that it checked by default, but till you haven't read the documentation about this behavior - it's dangerous to jump into conclusion